src/rdiffbackup/actions/__init__.py

```diff
@@ -466,14 +469,18 @@ def connect(self):
             Security.initialize(self.get_security_class(), cmdpairs)
             self.connected_locations = list(
                 map(SetConnections.get_connected_rpath, cmdpairs))
+
+            # if a connection is None, it's an error
+            for conn, loc in zip(self.connected_locations,
+                                 self.values.locations):
+                if conn is None:
+                    log.Log("Location '{lo}' couldn't be connected.".format(
+                        lo=loc), log.ERROR)
+                    self.conn_status = Globals.RET_CODE_ERR
         else:
             Security.initialize(self.get_security_class(), [])
             self.connected_locations = []
 
-        # once the connection is set, we can define "now" as being the current
-        # time, unless the user defined a fixed a current time.
-        Time.set_current_time(self.values.current_time)
```

Comment:
Since you move that away. You could also update the comments to remove 
```
Defines the current time as being the time of a potentially upcoming
        backup.
```

Reply:
Well spotted, it was also worth a comment in the git commit message.